src/ui/public/directives/tilemap_settings.js

```diff
@@ -0,0 +1,41 @@
+import uiModules from 'ui/modules';
+import _ from 'lodash';
+import marked from 'marked';
+marked.setOptions({
+  gfm: true, // Github-flavored markdown
+  sanitize: true // Sanitize HTML tags
+});
+
+uiModules.get('kibana')
+  .service('tilemapSettings', function (config, tilemap, $sanitize) {
```

Comment:
This seems to be in the wrong place.  It's under a "directives" directory, but it's a service.  If possible, let's keep this inside visualize itself.